src/convert_handle.hpp

```diff
@@ -0,0 +1,21 @@
+//
+//  convert_handle.hpp
+//
+
+#ifndef VG_CONVERT_HANDLE_HPP_INCLUDED
+#define VG_CONVERT_HANDLE_HPP_INCLUDED
+
+#include "handle.hpp"
+#include "vg.hpp"
+
+namespace vg {
+    using namespace std;
+    // Takes in a pointer to a HandleGraph and converts it to a MutableHandleGraph graph.
+    void convert_handle_graph(const HandleGraph* converting, MutableHandleGraph* converted);
+    
+    // Change paths to a mutable path.
+    void convert_path_handle_graph(const PathHandleGraph* converting, MutablePathDeletableHandleGraph* converted);
```

Comment:
I think you should be able to get away with a `MutablePathMutableHandleGraph`, right? You don't actually use the delete methods.

Reply:
I don't think vg, HashGraph, and PackedGraph have `MutablePathMutableHandleGraph`, I only see `MutablePathDeletableHandleGraph`

Reply:
`MutablePathDeletableHandleGraph` inherits from `MutablePathMutableHandleGraph`, so that's fine. The point of the distinction is to allow graph implementations that can be mutably "constructed" but don't support efficient delete operations. In fact, this is exactly the sort of use case we had in mind for `MutablePathMutableHandleGraph`.